README.md

```diff
@@ -941,11 +943,18 @@ INFO     Installing cat/fastq
 INFO     Downloaded 3 files to ./modules/nf-core/software/cat/fastq
 ```
 
-Use the `--tool` flat to specify a module name on the command line instead of using the cli prompt.
+You can pass the module name as a optional argument to `nf-core modules install` instead of using the cli prompt, eg: `nf-core modules install fastqc`.
```

Comment:
```suggestion
You can pass the module name as an optional argument to `nf-core modules install` instead of using the cli prompt, eg: `nf-core modules install fastqc`.
```